file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
+ *  - ::directory?:: Returns whether the file at the given path is a diretory.
+ *  - ::executable?:: Returns whether the file at the given path is executable
+ *                    by the effective user and group of the current process.
+ *  - ::executable_real?:: Returns whether the file at the given path is executable
+ *                         by the real user and group of the current process.
+ *  - ::exist?:: Returns whether the file at the given path exists.
+ *  - ::file?:: Returns whether the file at the given path is a file.
+ *  - ::ftype:: Returns a string giving the type of the file at the given path.
+ *  - ::grpowned?:: Returns whether the effective group of the current process
+ *                  owns the file at the given path.
+ *  - ::identical?:: Returns whether the files at two given paths are identical.
+ *  - ::owned?:: Returns whether the effective user of the current process
+ *               owns the file at the given path.
+ *  - ::pipe?:: Returns whether the file at the given path is a pipe.
+ *  - ::readable?:: Returns whether the file at the given path is readable
+ *                  by the effective user and group of the current process.
+ *  - ::readable_real?:: Returns whether the file at the given path is readable
+ *                       by the real user and group of the current process.
+ *  - ::setgid?:: Returns whether the setgid bit is set for the file at the given path.
+ *  - ::setuid?:: Returns whether the setuid bit is set for the file at the given path.
+ *  - ::socket?:: Returns whether the file at the given path is a socket.
+ *  - ::stat:: Returns the File::Stat object for the file at the given path.
+ *  - ::sticky?:: Returns whether the file at the given path has its sticky bit set.
+ *  - ::umask:: Returns the umask value for the current process.
+ *  - ::world_readable?:: Returns whether the file at the given path is readable
+ *                        by others.
+ *  - ::world_writable?:: Returns whether the file at the given path is writable
+ *                        by others.
+ *  - ::writable?:: Returns whether the file at the given path is writable by the current process.
+ *  - ::writable?:: Returns whether the file at the given path is writable
+ *                  by the effective user and group of the current process.
+ *  - ::writable_real?:: Returns whether the file at the given path is writable
+ *                       by the real user and group of the current process.
+ *
+ *  _Contents_
+ *
+ *  - ::empty? (aliasd as ::zero?):: Returns whether the file at the given path
+ *                                   exists and is empty.
+ *  - ::size:: Returns the size (bytes) of the file at the given path.
+ *  - ::size?:: Returns +nil+ if there is no file at the given path,
+ *              or if that file is empty; otherwise returns the file size (bytes).
+ *  - #size:: Returns the size (bytes) of +self+.
+ *
+ *  === Converting
```

Comment:
Methods in the `Converting` section should be moved to `Paths`, since they deal with path string manipulation.